tests/func-tests/observability_controller_test.go

```diff
@@ -12,17 +12,22 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	"github.com/kubevirt/hyperconverged-cluster-operator/controllers/observability"
+	"github.com/kubevirt/hyperconverged-cluster-operator/pkg/alertmanager"
```

Comment:
Can we avoid using the production code in the functional tests? Not sure it's the best practice.

Reply:
this is a package to communicate with the Alertmanager API, this could as well be like an external package, so IMO I think this should be fine to use

Reply:
I also meant the observability package. but ok